ods_ci/tests/Resources/Files/data-science-pipelines-reconciliation.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: sample
+spec:
+  objectStorage:
+    externalStorage:
+      bucket: rhods-dsp-dev
+      host: s3.amazonaws.com
+      s3CredentialsSecret:
+        accessKey: k8saccesskey
+        secretKey: k8ssecretkey
+        secretName: aws-bucket-creds
+      scheme: https
+  # Optional
+  mlpipelineUI:
```

Comment:
Do you need  mlpipelineUI? Can you delete it if not?

Reply:
Easy to debug if needed

Reply:
That pod won't be available.